plugin/PDFView.qml

```diff
@@ -132,6 +136,35 @@ SilicaFlickable {
         linkColor: Theme.highlightColor;
         pagePlaceholderColor: Theme.highlightColor;
 
+        onPageLayoutChanged: if (rememberPositionConfig.value) restorePageSettings()
```

Comment:
For the place of the code, *i.e.* PDFView or PDFDocumentPage, I hesitated much before deciding for PDFView because the actual values we want to save / restore are indeed there. If I put the save / restore functions in the PDFDocumentPage instance, we'll need accessors for the pageRectangle() function, the pageLayout signal and for the contentX and contentY properties. That finally decided me to put the code in PDFView, even if I agree that a dumber PDFView would have been desirable from a reusability point of view.

Besides, if we consider that's better to have a simpler PDFView object, I can figure out how to move the save / restore stuff to PDFDocumentPage.